<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>Field</class>
  <widget class="QWidget" name="Field">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>318</width>
        <height>260</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string notr="true">Form</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="3">
        <spacer name="verticalSpacer_2">
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
            <size>
              <width>20</width>
              <height>128</height>
            </size>
          </property>
        </spacer>
      </item>
      <item row="2" column="0">
        <spacer name="horizontalSpacer">
          <property name="orientation">
            <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
            <size>
              <width>177</width>
              <height>20</height>
            </size>
          </property>
        </spacer>
      </item>
      <item row="2" column="5">
        <spacer name="horizontalSpacer_2">
          <property name="orientation">
            <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
            <size>
              <width>177</width>
              <height>20</height>
            </size>
          </property>
        </spacer>
      </item>
      <item row="3" column="3">
        <spacer name="verticalSpacer">
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
            <size>
              <width>20</width>
              <height>128</height>
            </size>
          </property>
        </spacer>
      </item>
      <item row="2" column="3">
        <widget class="QWidget" name="centWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
              <widget class="QTabWidget" name="tabWidget">
                <property name="tabPosition">
                  <enum>QTabWidget::North</enum>
                </property>
                <property name="currentIndex">
                  <number>0</number>
                </property>
                <widget class="QWidget" name="tab_3">
                  <attribute name="title">
                    <string>Game</string>
                  </attribute>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                    <item>
                      <layout class="QVBoxLayout" name="center">
                        <item>
                          <layout class="QVBoxLayout" name="verticalLayout">
                            <item>
                              <widget class="QWidget" name="wEstate" native="true">
                                <layout class="QHBoxLayout" name="horizontalLayout_2">
                                  <item>
                                    <widget class="QLabel" name="lEstate">
                                      <property name="text">
                                        <string>Estate</string>
                                      </property>
                                    </widget>
                                  </item>
                                  <item>
                                    <spacer name="horizontalSpacer_5">
                                      <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                      </property>
                                      <property name="sizeHint" stdset="0">
                                        <size>
                                          <width>40</width>
                                          <height>20</height>
                                        </size>
                                      </property>
                                    </spacer>
                                  </item>
                                  <item>
                                    <widget class="QLabel" name="lPrice">
                                      <property name="text">
                                        <string>Price</string>
                                      </property>
                                    </widget>
                                  </item>
                                </layout>
                              </widget>
                            </item>
                          </layout>
                        </item>
                      </layout>
                    </item>
                    <item>
                      <widget class="QFrame" name="fButtons">
                        <property name="frameShape">
                          <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property name="frameShadow">
                          <enum>QFrame::Raised</enum>
                        </property>
                        <layout class="QHBoxLayout" name="horizontalLayout_3">
                          <item>
                            <layout class="QHBoxLayout" name="buttons">
                              <item>
                                <spacer name="horizontalSpacer_3">
                                  <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                  </property>
                                  <property name="sizeHint" stdset="0">
                                    <size>
                                      <width>40</width>
                                      <height>20</height>
                                    </size>
                                  </property>
                                </spacer>
                              </item>
                              <item>
                                <spacer name="horizontalSpacer_4">
                                  <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                  </property>
                                  <property name="sizeHint" stdset="0">
                                    <size>
                                      <width>40</width>
                                      <height>20</height>
                                    </size>
                                  </property>
                                </spacer>
                              </item>
                            </layout>
                          </item>
                        </layout>
                      </widget>
                    </item>
                  </layout>
                </widget>
              </widget>
            </item>
            <item>
              <widget class="QWidget" name="wSearch" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <property name="margin">
                    <number>0</number>
                  </property>
                  <item>
                    <widget class="QPushButton" name="bHideSearch">
                      <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                        </sizepolicy>
                      </property>
                      <property name="maximumSize">
                        <size>
                          <width>16</width>
                          <height>16</height>
                        </size>
                      </property>
                      <property name="text">
                        <string notr="true"/>
                      </property>
                      <property name="icon">
                        <iconset resource="../../../ressources.qrc">
                        <normaloff>:/images/tools/removetrade.png</normaloff>:/images/tools/removetrade.png</iconset>
                      </property>
                      <property name="flat">
                        <bool>true</bool>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QLabel" name="label">
                      <property name="text">
                        <string>Search for estate:</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="QLineEdit" name="tfsearch"/>
                  </item>
                </layout>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <resources>
    <include location="../../../ressources.qrc"/>
  </resources>
  <connections>
    <connection>
      <sender>bHideSearch</sender>
      <signal>clicked()</signal>
      <receiver>wSearch</receiver>
      <slot>hide()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>99</x>
          <y>159</y>
        </hint>
        <hint type="destinationlabel">
          <x>168</x>
          <y>159</y>
        </hint>
      </hints>
    </connection>
    <connection>
      <sender>bHideSearch</sender>
      <signal>clicked()</signal>
      <receiver>tfsearch</receiver>
      <slot>clear()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>89</x>
          <y>187</y>
        </hint>
        <hint type="destinationlabel">
          <x>218</x>
          <y>188</y>
        </hint>
      </hints>
    </connection>
  </connections>
</ui>
